akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileUploadDirectives.scala

```diff
@@ -58,6 +59,37 @@ trait FileUploadDirectives {
     }
 
   /**
+   * Streams the bytes of the file submitted using multipart with the given field name into temporary files on disk.
+   * If there is an error writing to disk the request will be failed with the thrown exception, if there is no such
+   * field the request will be rejected. Stored files are cleaned up on exit but not on failure.
+   *
+   * @group fileupload
+   */
+  def uploadedFiles(fieldName: String): Directive1[immutable.Seq[(FileInfo, File)]] =
```

Comment:
This needs a name which better describes what it does (maybe `storeUploadedFilesToTempFiles`? better ideas?). It should have a parameter to specify the directory to put the temporary files into.